src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -828,6 +829,80 @@ public static BinaryOperandsKind GetBinaryOperandsKind(BinaryOperationKind kind)
         }
     }
 
+    public interface IConversion
```

Comment:
I understand you argument and simply responding to your request to explain my proposal in details.